environments/staging-phase-one/group_vars/all/vars.yml

```diff
@@ -22,11 +22,11 @@ publishing_port: 6550
 authoring_host: staging00.cnx.org
 authoring_port: 6420
 
-zope_domain: legacy-staging00.cnx.org
-arclishing_domain: archive-staging00.cnx.org
-frontend_domain: staging00.cnx.org
+zope_domain: legacy-staging.cnx.org
+arclishing_domain: archive-staging.cnx.org
+frontend_domain: staging.cnx.org
 accounts_domain: accounts.openstax.org
-tutor_domain: tutor.openstax.org
+tutor_domain: tutor-staging.openstax.org
```

Comment:
I’m not exactly in favor of this kind of arrangement. I think the cnx (and tutor) staging environment should point at the production services that are out of their scope. Otherwise, we end up tightly coupling changes to services that may or may not be deployed (deadlock scenario). That means when two or more services need deployed at the same time, which isn’t usually a good idea because there is a lot more risk involved, we will continually run into chicken and egg issues. In this case, cnx comes before tutor, but that may not always be the case.